pkg/federated/README.md

```diff
@@ -0,0 +1,20 @@
+# Federated Search
+
+Use federated search to query and combine results from multiple Red Hat Advanced Cluster Management.
+
+## Pre-requisites  
+1. Multicluster Global Hub Operator 1.0 or later.  
+2. Red Hat Advanced Cluster Management 2.10 or later.  
+
+## Setup
+Execute the script at `./setup.sh` to configure Global Search on the Global Hub cluster.  
```

Comment:
Add required RBAC permissions to execute this script.

Reply:
resolved with commit https://github.com/stolostron/search-v2-api/pull/189/commits/c1c6f8aea9e70c6b531be736bd4f11a669222d98